### ISSUE TITLE
Work on paths with spaces

### ISSUE THREAD

**Issue by richfitz** [dide-tools/didewin#9]
```
setwd("Q:/cluster_test/path with spaces")
didewin::didewin_config_global(credentials="rfitzjoh",
                               cluster="fi--didemrchnb")
ctx <- context::context_save("contexts", packages=NULL, sources=NULL)
context::context_log_start()
obj <- didewin::queue_didewin(ctx)
```

gives

```
Loading context f413d5b3eb34996d77ffb38f680ad15d
[ context   ]  f413d5b3eb34996d77ffb38f680ad15d
[ lib       ]  contexts/R/x86_64-w64-mingw32/3.3.0
[ library   ]
[ namespace ]
[ source    ]
[ cross     ]  checking available packages
[ cross     ]  Downloading binary packages for: crayon, curl, digest, drat, memoise, R6, storr, uuid
trying URL 'https://cran.rstudio.com/bin/windows/contrib/3.2/crayon_1.3.1.zip'
Content type 'application/zip' length 606260 bytes (592 KB)
downloaded 592 KB
...
[ cross     ]  Cross installing source packages for: context
trying URL 'https://richfitz.github.io/drat/src/contrib/context_0.0.7.tar.gz'
Content type 'application/octet-stream' length 41779 bytes (40 KB)
downloaded 40 KB

Warning: invalid package 'with'
Warning: invalid package 'spaces/contexts/R/x86_64-w64-mingw32/3.2.4'
Error: ERROR: cannot cd to directory 'Q:/cluster_test/path'
Error in FUN(X[[i]], ...) : Command failed (code: 1)
In addition: Warning message:
running command '"C:/PROGRA~1/R/R-33~1.0/bin/R" R_LIBS_USER=Q:/cluster_test/path with spaces/contexts/R/x86_64-w64-mingw32/3.2.4;Q:/cluster_test/path with spaces/contexts/R/x86_64-w64-mingw32/3.3.0;C:/Users/rfitzjoh/Documents/R/win-library/3.3;C:/Program Files/R/R-3.3.0/library CYGWIN=nodosfilewarning CMD INSTALL --no-test-load --library=Q:/cluster_test/path with spaces/contexts/R/x86_64-w64-mingw32/3.2.4 C:\Users\rfitzjoh\AppData\Local\Temp\Rtmpiq85SP\file8e030676c43b6\context' had status 1 
```

**Comment by richfitz** [comment]
traceback:

```
10: stop(sprintf("Command failed (code: %d)", ok))
9: FUN(X[[i]], ...)
8: lapply(packages_src, cross_install_package, pkgs_src, lib, FALSE, 
       platform)
7: cross_install_packages(lib, platform, r_version, repos, "context")
6: cross_install_bootstrap(lib, platform, r_version, root)
5: context::cross_install_context(path_lib, "windows", r_version_2, 
       context)
4: initialise_packages(self)
3: public_bind_env$initialize(...)
2: .R6_queue_didewin$new(context, config, login, rtools)
1: didewin::queue_didewin(ctx)
```

**Comment by richfitz** [comment]
Fixed via a093508 and dide-tools/context@1b29030
